pkgs/top-level/perl-packages.nix

```diff
@@ -14404,6 +14408,21 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  TextFormat = buildPerlPackage rec {
+    name = "Text-Format-0.60";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/S/SH/SHLOMIF/${name}.tar.gz";
+      sha256 = "664f313570604624ff9e1fc9b26b6d04e06897b3e4eac83089fc0905a692a2b8";
+    };
+    buildInputs = [ ModuleBuild ];
+    meta = {
+      homepage = http://www.shlomifish.org/open-source/projects/Text-Format/;
+      description = "Format text";
+      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
+      maintainer = with maintainers; [ bcdarwin ];
```

Comment:
`with stdenv.lib.maintainers;`

Reply:
perl-packages has stdenv.lib.maintainers in scope

Reply:
My bad, thanks.